src/components/Modal/components/Dialog/tests/Dialog.test.tsx

```diff
@@ -16,23 +15,26 @@ describe('<Dialog>', () => {
   });
 
   it('sets CloseKeypressListener when `in` is true', () => {
-    const listener = mountWithAppProvider(
+    const listener = mountWithApp(
       <Dialog labelledBy="test" onClose={jest.fn()} in>
         something
       </Dialog>,
-    ).find(KeypressListener);
+    );
 
-    expect(listener.exists()).toBe(true);
+    expect(listener).toContainReactComponent(KeypressListener);
   });
 
   it('triggers an onEntered prop', () => {
-    const dialog = mountWithAppProvider(
-      <Dialog labelledBy="test" onClose={jest.fn()} onEntered={jest.fn()}>
+    const spy = jest.fn();
+
+    const dialog = mountWithApp(
+      <Dialog labelledBy="test" onClose={jest.fn()} onEntered={spy}>
```

Comment:
Does `onClose` need to be mocked here? Could this be the following (just to match with the patterns in other tests)?
```js
onClose={noop}
```

Adding a 
```js
function noop() {}
```
To the end of the test file?